filebeat/docs/filebeat-modules-options.asciidoc

```diff
@@ -1,3 +1,5 @@
+:modulename: apache2 mysql
```

Comment:
Not introduced here but I wonder why we specify these two here.

Reply:
I'm using an asciidoc attribute to resolve the module name(s) used in include/enable-modules-command.asciidoc. (After adding the platform-specific commands, I realized that it would be easier to maintain the content in a single place, so I abstracted some of the content into shared files under `/include`.) In this particular topic, we want to show an example that enables two modules, so I set the `modulename` to include two modules. 

I should probably add comments to the files to indicate how the attributes are used, but that gets complicated with asciidoc because comments _sometimes_ break things like lists and parts.

Reply:
Thanks for the background info.